services/token_update.proto

```diff
@@ -142,4 +142,15 @@ message TokenUpdateTransactionBody {
      * transaction will resolve to TOKEN_HAS_NO_PAUSE_KEY
      */
     Key pause_key = 15;
+
+    /**
+     * Metadata of the created token definition
+     */
+    google.protobuf.BytesValue metadata = 16;
```

Comment:
Why is this google.protobuf.BytesValue instead of bytes?

Reply:
I believe google.protobuf.BytesValue was chosen for metadata in order to make this field optional and easier to extend this transaction type later

Reply:
It's `bytes` in token_create but `google.protobuf.BytesValue` in token_update. I'm not sure which is more appropriate but at a minimum they should match.

Reply:
We have always used wrappers in `*Update` transaction bodies to differentiate between the value being set or not.  Just like `StringValue` for memo in `token_update` and `string`  in token_create.